docs/ops.html

```diff
@@ -229,6 +229,63 @@ <h4><a id="basic_ops_consumer_group" href="#basic_ops_consumer_group">Managing C
   Deletion of requested consumer groups ('my-group', 'my-other-group') was successful.
   </pre>
 
+  <p>
+  To reset offsets of a consumer group to latest offset, first make sure the instances are inactive, then:
```

Comment:
nit: "to _the_ latest offset"

Also, "instances" is a bit vague. We could say "consumer instances" or maybe just "consumers."

Reply:
Ack.